tests/acceptance/features/apiSharingNg/shareInvitations.feature

```diff
@@ -1111,3 +1111,29 @@ Feature: Send a sharing invitations
       | Co Owner         | folder        | FolderToShare  |
       | Uploader         | folder        | FolderToShare  |
       | Manager          | folder        | FolderToShare  |
+
+
+  Scenario Outline: send share invitation to normal user
```

Comment:
we should have this scenario already covered, right?

Reply:
This scenario checks whether the `sharee` receives the share or not. A new API endpoint `shareWithMe` is used for the `then` step. Previously added tests check for request and response only.

Reply:
Then maybe we can just extend the scenarios above with one Then step